include/logger.h

```diff
@@ -8,14 +8,22 @@
 
 // * Log record : each line of the log
 // * log format : tuple like <Date, Name, Message>
-// * log entity : Date, Name, Message
+// * log property : Date, Name, Message
 #define LOG(severity)                                                          \
   logger::Logger::getInstance().log<logger::severity>(                         \
-      {__func__, __FILE__, __LINE__})
+      {__FILE__, __func__, __LINE__})
+
+
+std::string hex(intptr_t v) {
+  std::stringstream ss;
```

Comment:
Might want to look at alternative implementations of this - I recall that instantiating a stringstream is pretty expensive. Can we reuse somehow?